one-to-one-scheduler/one-to-one-scheduler.gs

```diff
@@ -626,43 +625,161 @@ function sendInvitesToMentors() {
+  let setupResult = initSetup();
+  if (!setupResult.success) {
     return;
   }
 
-  if (!confirmCalendar(calendar)) {
-    showInfo(TITLE_ERROR, MSG_CALENDAR_REJECTED);
-    return;
+  let noMeetingRows = [];
+  let invalidMeetings = [];
+  let noRecordingLinks = [];
+  let noChatLinks = [];
+  let numSuccess = 0;
+  let numSkipped = 0;
+  let calendarId = setupResult.calendar.getId();
+
+  for (let rowIndex = 0; rowIndex < setupResult.oneToOneData.length; ++rowIndex) {
+    let rowNumber = rowIndex + DATA_BEGIN_ROW;
+    let opsRequestFlag = parseInt(setupResult.activeSheet.getRange(rowNumber, OPS_REQUEST_INDEX + 1).getValue());
+    if (opsRequestFlag !== FLAG_SEND_INVITE) {
+      // The row should be skipped
+      ++numSkipped;
+      continue;
+    }
+
+    let meetIdCell = setupResult.activeSheet.getRange(rowNumber, MEETING_ID_INDEX + 1);
+    let meetId = meetIdCell.getValue().trim();
+    if (meetId === "") {
+      // Meeting is not set
+      noMeetingRows.push(rowNumber);
+      continue;
+    }
+
+    let event = getEvent(setupResult.calendar, meetId);
+    if (!event) {
+      invalidMeetings.push(rowNumber);
+      continue;
+    }
+
+    let eventResource = Calendar.Events.get(calendarId, meetId);
+    if (!eventResource) {
+      console.log("Event could not be retrieved - " + calendarId + ", " + meetId);
+      invalidMeetings.push(rowNumber);
+      continue;
+    }
+
+    let eventAttachments = eventResource.attachments;
+    
+    let recordingLinkCell = setupResult.activeSheet.getRange(rowNumber, RECORDING_LINK_INDEX + 1);
+    let chatLinkCell = setupResult.activeSheet.getRange(rowNumber, CHAT_LINK_INDEX + 1);
+
+    if (!eventAttachments) {
+      recordingLinkCell.setValue(NOT_FOUND);
+      chatLinkCell.setValue(NOT_FOUND);
+      noRecordingLinks.push(rowNumber);
+      noChatLinks.push(rowNumber);
+      continue;
+    } else {
+      let videoFound = false;
+      let chatFound = false;
+      for (let i = 0; i < eventAttachments.length; ++i) {
+        let currentAttachment = eventAttachments[i];
+        let linkUrl = currentAttachment.fileUrl;
+        if (linkUrl) {
+          let queryIndex = linkUrl.indexOf(QUERY_DELIMITER);
+          if (queryIndex > 0) {
+            linkUrl = linkUrl.substring(0,queryIndex);
+          }
+          if (currentAttachment.mimeType === RECORDING_MIME_TYPE) {
+            videoFound = true;
+            recordingLinkCell.setValue(linkUrl);
+          } else if (currentAttachment.mimeType === CHAT_MIME_TYPE) {
+            chatFound = true;
+            chatLinkCell.setValue(linkUrl);
+          }
+        }
+      }
+      if (videoFound && chatFound) {
+        ++numSuccess;
+        continue;
+      }
+      if (!videoFound && chatFound) {
+        recordingLinkCell.setValue(NOT_FOUND);
+        noRecordingLinks.push(rowNumber);
+      } else if (videoFound && !chatFound) {
+        chatLinkCell.setValue(NOT_FOUND);
+        noChatLinks.push(rowNumber);
+      }
```

Comment:
It is sufficient to check for `!videoFound`. Also, no need of `else if`.
```js
      if (!videoFound) {
        recordingLinkCell.setValue(NOT_FOUND);
        noRecordingLinks.push(rowNumber);
      }
      if (!chatFound) {
        chatLinkCell.setValue(NOT_FOUND);
        noChatLinks.push(rowNumber);
      }
```

Reply:
Updated the Requirement